amf-cli/shared/src/test/resources/validations/reports/async20/anypointmq-invalid-version.report

```diff
@@ -1,22 +1,22 @@
 ModelId: file://amf-cli/shared/src/test/resources/validations/async20/validations/anypointmq-invalid-version.yaml
-Profile:
+Profile: 
 Conforms: false
 Number of results: 2
 
 Level: Violation
 
 - Constraint: http://a.ml/vocabularies/amf/parser#unsupported-binding-version
-  Message: Version 0.2.0 is not supported in an AnypointMQ
+  Message: Version 0.2.0 is not supported in a AnypointMQ Binding
   Severity: Violation
-  Target: AnypointMQ
-  Property:
-  Range: [(8,17)-(12,0)]
+  Target: file://amf-cli/shared/src/test/resources/validations/async20/validations/anypointmq-invalid-version.yaml#/async-api/endpoint/some-channel/channel-bindings/bindings/anypointmq-channel
+  Property: 
+  Range: 
   Location: file://amf-cli/shared/src/test/resources/validations/async20/validations/anypointmq-invalid-version.yaml
 
 - Constraint: http://a.ml/vocabularies/amf/parser#unsupported-binding-version
-  Message: Version 1.3.0 is not supported in an AnypointMQ
+  Message: Version 1.3.0 is not supported in a AnypointMQ Binding
   Severity: Violation
-  Target: AnypointMQ
-  Property:
-  Range: [(16,21)-(24,0)]
+  Target: file://amf-cli/shared/src/test/resources/validations/async20/validations/anypointmq-invalid-version.yaml#/async-api/endpoint/some-channel/supportedOperation/subscribe/returns/resp/default-response/message-bindings/bindings/anypointmq-message
+  Property: 
+  Range: 
```

Comment:
Lexical information is disappearing here. Why? It should not be error messages without lexical

Reply:
Done, restored the lexical